app_test.go

```diff
@@ -15,7 +22,8 @@ import (
 )
 
 const (
-	ytDevToken = "password"
+	ytDevToken              = "password"
+	runTestsWithNewYtEnvVar = "RUN_TESTS_WITH_NEW_YT"
```

Comment:
Name of env var is confusing, sounds like we will run tests with a new version of ytsaurus or something.
Let's call it `REUSE_YT_CONTAINER` and by default it will not be set in gh-workflow.